modules/sdk/node-build/esbuild/getExternals.mjs

```diff
@@ -0,0 +1,5 @@
+export default function getExternals(globalImports) {
+	return [
+		...new Set(Object.values(globalImports).map(({webContextPath}) => `../../../${webContextPath}/*`))
```

Comment:
`...new Set(`, I'm shocked this works. Also.... why use a set when we already have an array? Maybe a js comment for why

Reply:
I use a `Set` to dedupe items. But yeah, I can add a comment to explain it :sweat_smile: .

Reply:
done